net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -53,27 +53,38 @@ src_configure() {
 
 src_install() {
 	cmake-utils_src_install
+
+	# config
+	insinto "/etc/i2pd"
+	doins "${S}/docs/i2pd.conf"
+	doins "${S}/debian/subscriptions.txt"
+	doins "${S}/debian/tunnels.conf"
+
+	# doc
 	dodoc README.md
+
+	# working directory
 	keepdir /var/lib/i2pd/
-	insinto "/var/lib/i2pd"
+	insinto /var/lib/i2pd
 	doins -r "${S}/contrib/certificates"
 	dosym /etc/i2pd/subscriptions.txt /var/lib/i2pd/subscriptions.txt
 	fowners "${I2PD_USER}:${I2PD_GROUP}" /var/lib/i2pd/
 	fperms 700 /var/lib/i2pd/
-	dodir "/etc/${PN}"
-	insinto "/etc/${PN}"
-	doins "${S}/docs/${PN}.conf"
-	doins "${S}/debian/subscriptions.txt"
-	doins "${S}/debian/tunnels.conf"
```

Comment:
Do these belong in /etc/i2pd?  I know above you just moved them, but shouldn't these be under /usr/share/doc/i2pd-${PV}?  Or does the daemon expect them to be under /etc/i2pd?